src/Plugins/Lmgtfy.php

```diff
@@ -0,0 +1,79 @@
+<?php declare(strict_types=1);
+
+namespace Room11\Jeeves\Plugins;
+
+use Room11\Jeeves\Chat\Client\ChatClient;
+use Room11\Jeeves\Chat\Message\Command;
+use Room11\Jeeves\System\PluginCommandEndpoint;
+use Room11\Jeeves\Chat\Client\MessageResolver;
+use Room11\Jeeves\Chat\Room\Room;
+
+class Lmgtfy extends BasePlugin
+{
+    const URL = 'http://lmgtfy.com/';
+    const USAGE = 'Usage: `!!lmgtfy [ <text> ]` / `!!lmgtfy [ <message URL> ]`';
+
+    private $chatClient;
+
+    public function __construct(ChatClient $chatClient, MessageResolver $messageResolver)
+    {
+        $this->messageResolver = $messageResolver;
+        $this->chatClient = $chatClient;
+    }
+
+    public function lmgtfy(Command $command)
+    {
+        $text = $command->getText();
+
+        if ((bool) preg_match('~^http://chat\.stackoverflow\.com/transcript/message/(\d+)(#\d+)?$~', $text)) {
+            $text = yield from $this->getRawMessage($command->getRoom(), $text);
```

Comment:
Using [`MessageResolver#resolveMessageText()`](https://github.com/Room-11/Jeeves/blob/master/src/Chat/Client/MessageResolver.php#L43) here directly is probably better in this case - it will result in fewer requests and avoids the `html_entity_decode()`, which can result in weird things happening in the case of complex messages e.g. those containing `inline code`.

`resolveMessageText()` returns the un-rendered markdown directly.